Update NGE Docs

PatrickRyanMS opened the pull request:
Add all new nodes, update node images, add new Debug Info to all nodes.

closes #1088

PatrickRyanMS committed:
Update NGE Docs

Add all new nodes, update node images, add new Debug Info to all nodes.

Popov72 reviewed (commented):
Great job, only a few comments!

PatrickRyanMS committed:
Update content/toolsAndResources/nge/ngeBlocks.md

Co-authored-by: Popov72 <github@evpopov.com>

PatrickRyanMS committed:
Updating camel case for links

Updated two descriptions for accuracy and fixed mismatched camel case on links to images.

PatrickRyanMS committed:
Updating file names

Changes to file names to correct camel case.

PatrickRyanMS commented:
Thanks for the corrections, @Popov72! I have made the corrections and updated the PR
